### PR TITLE
Add IWWG Charter 22/23 deliverable updates and links

### PR DESCRIPTION
**Pull request by Yonet**
Charter updates from. our working [document](https://docs.google.com/document/d/1FYsmyBxt4ntiGwvT6HRMlMX35L1G9bTTz8v2_WgydHc/edit)

### CODE REVIEW

**Comment by AdaRoseCannon** [comment]
Thank you so much!!!

**Comment by himorin** [comment]
(temporal) HTML diff between HEAD (of main) to f1d3cc4 :
https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fimmersive-web-wg-charter%2Fimmersive-web-wg-charter.html&doc2=https%3A%2F%2Fstorage.himor.in%2Ftemp-W3C%2Fimmersive-web-wg-charter-yonet-main-f1d3cc4.html

File: immersive-web-wg-charter.html [hunk 1]
```diff
@@ -1,717 +1,971 @@
...
+              target="_blank">WebXR Device
+              API</a></dt>
+          <dd>
+            <p>We will continue to maintain the core WebXR Device API but we don’t intend to add new features or make
+              substantial
+              changes.</p>
+            <p>This specification defines support for accessing virtual reality (VR) and augmented reality (AR) devices,
+              including sensors and head-mounted displays, on the Web.</p>
+            <p class="draft-status"><b>Draft state:</b> <a href="https://www.w3.org/standards/types#ED"
+                target="_blank">Editor's Draft</a></p>
```
Comment (himorin) [inline immersive-web-wg-charter.html:257]: I believe we should put the latest status "Working Draft" once we passed FPWD..
e.g. https://w3c.github.io/charter-drafts/pe-2020.html#deliverables

**Comment by himorin** [comment]
we can copy&paste from https://www.w3.org/groups/wg/immersive-web/deliverables for lines on exclusion draft (and related)

**Comment by himorin** [comment]
- Raw camera access
- navigation
- Real World Geometry
are still in CG, so we might need to group these three with a block, starting from `The Working Group also intends to work on the following Recommendation-Track deliverables, `?

**Comment by himorin** [comment]
@Yonet I can take an action to update details and links in each specification block, like link of editors/working draft and/or exclusion draft, after this PR merged. Please feel free to keep left them not checked/fixed for merging this PR.

**Comment by Yonet** [comment]
@himorin feel free to merge this draft. We will continue to work on it after the new year. I will try to get realistic completion dates by then. Thank you and happy new year!

File: immersive-web-wg-charter.html (hunk 1 above)
Comment (Yonet) [reply immersive-web-wg-charter.html]: Changed it to working draft. Thanks @himorin

**Comment by himorin** [comment]
> @himorin feel free to merge this draft. We will continue to work on it after the new year. I will try to get realistic completion dates by then. Thank you and happy new year!

Thank you. I'll merge this shortly, and work on lines for details of specs listed.

**Pull request merged**